backend/src/Altinn.Authorizationadmin/Altinn.AuthorizationAdmin.Core/Models/Delegation.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System.Text.Json.Serialization;
+
+namespace Altinn.AuthorizationAdmin.Core.Models
+{
+    /// <summary>
+    /// This model describes a delegation. A delegation is an action that says if a resource is delegated to you or you have delegated a resource to another person/org
+    /// </summary>
+    public class Delegation
+    {
+        /// <summary>
+        /// Gets or sets the identifier of the resource in a delegation
+        /// </summary>
+        [JsonPropertyName("resourceid")]
+        public string ResourceId { get; set; }
+
+        /// <summary>
+        /// Gets or sets the title of the resource in a delegation
+        /// </summary>
+        [JsonPropertyName("resourcetitle")]
+        public Dictionary<string, string> ResourceTitle { get; set; }
```

Comment:
Still not sure why the need to have ResourceId and ResourceTitle duplicated on the ResourceDelegation model and on the Delegation model.

Depending on what this model is to be used for and displayed in the frontend, I assume we also will need more information from the delegationchange model? Time of delegation etc?

Reply:
This is because i had a join with resourceregistry table. The join is removed and the data is fetched via resourceregistry component